kotlin-data-wedge-lib/src/main/java/dk/gls/kdw/model/barcode/composite/CompositeAB.kt

```diff
@@ -0,0 +1,20 @@
+package dk.gls.kdw.model.barcode.composite
+
+import android.os.Bundle
+import dev.ahmedmourad.bundlizer.Bundlizer
+import dk.gls.kdw.model.barcode.model.UCCLinkMode
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+@Serializable
+data class CompositeAB(
+    @SerialName("decoder_composite_ab")
+    val enabled: Boolean = true,
+    /**
+     * Select an option for linking UPC barcodes with a 2D barcode during transmission as if they were one barcode
```

Comment:
Says UPC in comments but is named ucc - are they the same?

Reply:
Actually a good question. But this is copied from the official documentation. As I have based the entire library and the documentation on that, I think we have too just trust it.